electronics/capacitor_equivalence.py

```diff
@@ -0,0 +1,57 @@
+# https://farside.ph.utexas.edu/teaching/316/lectures/node46.html
+
+from __future__ import annotations
+
+
+def capacitor_parallel(capacitors: list[float]) -> float:
+    """
+    Ceq = C1 + C2 + ... + Cn
+    Calculate the equivalent resistance for any number of capacitors in parallel.
+    >>> capacitor_parallel([5.71389, 12, 3])
+    20.71389
+    >>> capacitor_parallel([5.71389, 12, -3])
+    Traceback (most recent call last):
+        ...
+    ValueError: Capacitor at index 2 has a negative value!
+    """
+    sum_c = 0.00
+    index = 0
+    for capacitor in capacitors:
+        sum_c += capacitor
+        if capacitor < 0:
+            msg = f"Capacitor at index {index} has a negative value!"
+            raise ValueError(msg)
+        index += 1
```

Comment:
```suggestion
    for index, capacitor in enumerate(capacitors):
        if capacitor < 0:
            msg = f"Capacitor at index {index} has a negative value!"
            raise ValueError(msg)
        sum_c += capacitor
```
Use `enumerate` to iterate over a list with the indices